src/components/project/settings/EventGroupingPattern.vue

```diff
@@ -0,0 +1,295 @@
+<template>
+  <div class="projects-patterns-settings-page">
+    <div class="settings-window-page__title">
```

Comment:
as i see, in the project bem pattern is used for classnames. So, the class should be `projects-patterns-settings-page__title`